### PR TITLE
Added more birthday splashes

### PR DESCRIPTION
**Pull request by leha-code**

### CODE REVIEW

File: src/mcpil.py [hunk 1]
```diff
@@ -534,6 +538,11 @@ def main():
     window.title('MCPIL')
     window.geometry('512x400')
     window.resizable(True, True)
+    
+        if not time_now.day == 1 and not time_now.month == 4: #The month and the day isn't the Aprils's fools day
```
Comment (TheBrokenRail) [inline src/mcpil.py:542]: This is in another PR.

File: src/mcpil.py [hunk 2]
```diff
@@ -331,14 +331,18 @@ def play_tab(parent):
...
-    elif today == 2 and today.day == 20:
+    elif today.month == 2 and today.day == 20:
         splash = "Happy Birthday Boba"
-    elif today == 7 and today.day == 15:
+    elif today.month == 7 and today.day == 15:
         splash = "Happy Birthday RPiNews!"
+    elif today.month == 9 and today.day == 13:
+        splash = "Happy B-Day gamer4life1"
+    elif today.month == 7 and today.day == 18:
+        splash = "BED.HEAL(\"TheBoyLeastLikelyTo\");"
```
Comment (TheBrokenRail) [inline src/mcpil.py:345]: Who is this? What is the criteria for who is added here? Because we can't add everyone.

**Review by nobody5050 — CHANGES_REQUESTED** [review]
this is mostly a useless pr.

**Comment by leha-code** [comment]
I added least likel and gamercat

**Comment by leha-code** [comment]
They asked for it

**Comment by leha-code** [comment]
Ill change ths evening everything that is needed

**Comment by leha-code** [comment]
I wont add anyone else
That. Was the last. Sorry typing froman ebook is hell

**Comment by StealthHydra179** [comment]
1. Inactive 2. Requested changes not made 3. PR not needed

**Pull request closed without merging**